### ISSUE TITLE
Y-axis tick labels not updating

### ISSUE THREAD

**Issue by travis23** [rosenbrockc/live-serial#20]
The y-axis tick are not updating as the data scales

```
[sensor.prox]
dtype=int, float
port=COM4
label=Proximity
value_index=2

[port.COM4]
port_baud=115200
port_timeout=0.3
port_encoding=ASCII
# port_delimeter=\s

[global]
wait=10
buffertime=5
logdir=path
```

**Comment by rosenbrockc** [comment]
On Windows machines, we can optimize the animation speed using `blit=True` in `matplotlib`. However, this prevents the axes from updating their labels. We could put a routine in to check every frame whether we went out of range and then manually redraw the axes, but that seems unnecessary for now. Instead, I just made the default behavior to have `blit=False` for Windows (it is the default for Unix too). Since machines are so fast these days, it doesn't really make much of an issue. If you get lots of subplots (more than 6) you might start to notice something.
